wagtail/project_template/home/migrations/0002_create_homepage.py

```diff
@@ -49,6 +49,8 @@ def remove_homepage(apps, schema_editor):
 
 class Migration(migrations.Migration):
 
+    run_before = ('wagtailcore', '0053_locale_model')
```

Comment:
`run_before` should be an [array of tuples](https://docs.djangoproject.com/en/3.1/howto/writing-migrations/#controlling-the-order-of-migrations). As it is now, Django will raise a `NodeNotFoundError` exception with a `nonexistent child node 'wagtailcore'` message.

```
run_before = [
    ('wagtailcore', '0053_locale_model'),
]
```

Reply:
Thank you for taking the time to review this. You are right. Should be a lesson not to make tweaks in a rush (busy schedule be damned) 🤦‍♂️